### PR TITLE
Fixes #1438 - Adding browserstack logo

### LINKED ISSUE #1438: add browserstack logo/blurb to site/readme
hi, the free browserstack account is coming up on expiration. a support contact made the request that in order to extend the free account, some credits should be visible on the site and/or readme.

I can work on this (target next couple of days).

### PR DESCRIPTION
**Pull request by cch5ng**
Browserstack marketing/#1438

### CODE REVIEW

**Comment by cch5ng** [comment]
I think I will try removing the browserstack logo from the README and add it to the homepage. in order to keep to the current organization, I think this might require adding a template under the home-page templates and this feels like a little bit of overkill but if wrong, it can be corrected afterward.

the good thing is that this would allow for testing the logo link earlier.

File: webcompat/templates/home-page/thanks.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<section class="wc-UISection">
+  <div class="wc-UIContent">
+    <h2 class="wc-Title wc-Title--l">
+      <a id="thanks">
+        Thanks
+      </a>
+    </h2>
+    <p class="wc-Thanks-p">Thanks to <a href="https://www.browserstack.com" target="_blank">BrowserStack</a> for providing the infrastructure that allows us to test in real browsers.
+    <div class="wc-Thanks-logo">
+      <a href="https://www.browserstack.com" target="_blank"><span class="wc-Browserstack"></span></a>
```
Comment (zoepage) [inline webcompat/templates/home-page/thanks.html]: Please add a title to the link :)

File: webcompat/static/css/development/components/threesteps.css [hunk 2]
```diff
@@ -29,6 +29,25 @@
...
 
+.wc-Browserstack {
+
+  width: 100%;
+  background-image:url(/img/browserstack_logo.svg);
+  background-size: cover;
+  height: 0;
+  padding: 0; /* reset */
+  padding-bottom: 22%;
+/*  width: 154px;*/
```
Comment (zoepage) [inline webcompat/static/css/development/components/threesteps.css]: Do we need those two comments? :)

**Comment by cch5ng** [comment]
hi @zoepage thanks! I made the requested changes. also cleaned up logo padding with better shorthand.

**Comment by zoepage** [comment]
@cch5ng Awesome, thank you :)
If you could squash the commits, we'd be good to go!

(Note: edited title of PR to make issue close automatically.)

**Comment by cch5ng** [comment]
> If you could squash the commits,
@zoepage done, thanks...

**Comment by zoepage** [comment]
LMTG! Thanks @cch5ng :)

**Comment by cch5ng** [comment]
hi @zoepage I can see all the source changes were merged but when I go to the webcompat.com site, I do not see the new section on the home page. do you think I missed something in the code changes? there is no new section from inspect element. but I could see the change ok when testing with a local web server... any ideas?

browserstack support had asked to place a text blurb and logo. I don't know if they'll nitpick over the logo placement since the README has a link now. but I should know the process to update page content for the future.

**Comment by miketaylr** [comment]
> I do not see the new section on the home page.

Sorry about that! I hadn't yet done a deploy, but it's live now.

**Comment by cch5ng** [comment]
thanks @miketaylr for the deploy. I let browserstack support know their changes requests are up and registered an account using the webcompat@gmail.com address.

**Comment by zoepage** [comment]
Thank you @cch5ng && @miketaylr :)

**Pull request merged**